jtravis/src/main/java/fr/inria/spirals/jtravis/helpers/AbstractHelper.java

```diff
@@ -109,6 +114,21 @@ protected String get(String url) throws IOException {
         return result;
     }
 
+    protected String getV3(String url) throws IOException {
+        Request request = this.requestBuilderV3(url).get().build();
```

Comment:
ok there is a misunderstanding here: I did not want to have access to the V2 and the V3 at the same time from the API. Then you should not duplicated the methods, but replace them. 
And even if I wanted to keep an access to V2, I don't want it like that: here the code is completely duplicated. If you need to keep both, add a boolean or an enum somewhere for qualifying the access to V3/V2 and use them throughout the calls.